hmpps-prisoner-search-indexer/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/events/HmppsDomainEventsEmitterIntTest.kt

```diff
@@ -530,47 +524,47 @@ class HmppsDomainEventsEmitterIntTest : IntegrationTestBase() {
    *
    * So this test checks that the prisoner event hash update is rolled back if sending the domain event fails.
    */
-  @Test
-  fun `e2e - should not update prisoner hash if there is an exception when sending the event`() {
-    recreatePrisoner(PrisonerBuilder(prisonerNumber = "A1239DD", bookingId = null))
-
-    val message = "/messages/offenderDetailsChanged.json".readResourceAsText().replace("A7089FD", "A1239DD")
-
-    // remember the prisoner event hash
-    val insertedPrisonerEventHash = prisonerHashRepository.findByIdOrNull("A1239DD")?.prisonerHash
-    assertThat(insertedPrisonerEventHash).isNotNull
-
-    // update the prisoner on ES BUT fail to send an event
-    doThrow(RuntimeException("Failed to send event")).whenever(publishQueueSqsClient)
-      .sendMessage(any<SendMessageRequest>())
-    prisonApi.stubFor(
-      get(urlEqualTo("/api/prisoner-search/offenders/A1239DD"))
-        .willReturn(
-          aResponse()
-            .withHeader("Content-Type", "application/json")
-            .withBody(
-              PrisonerBuilder(
-                prisonerNumber = "A1239DD",
-                bookingId = null,
-                firstName = "NEW_NAME",
-              ).toOffenderBooking(),
-            ),
-        ),
-    )
-    offenderQueueSqsClient.sendMessage(message)
-    await untilCallTo { getNumberOfMessagesCurrentlyOnEventQueue() } matches { it == 0 }
-    await untilAsserted { verify(publishQueueSqsClient).sendMessage(any<SendMessageRequest>()) }
-
-    // The prisoner hash update should have been rolled back
-    val prisonerEventHashAfterAttemptedUpdate =
-      prisonerHashRepository.findByIdOrNull("A1239DD")?.prisonerHash
-    assertThat(prisonerEventHashAfterAttemptedUpdate).isEqualTo(insertedPrisonerEventHash)
-  }
+//  @Test
+//  fun `e2e - should not update prisoner hash if there is an exception when sending the event`() {
```

Comment:
Is this a reminder to do the SNS retry work?

Reply:
Yes basically!